src/features/colinks/BuyOrSellCoLinks.tsx

```diff
@@ -53,23 +67,26 @@ export const BuyOrSellCoLinks = ({
 
   const [progress, setProgress] = useState('');
 
-  const { library, account } = useWeb3React();
+  const { chainId, account } = useWeb3React();
   const doWithCoLinksContract = useDoWithCoLinksContract();
 
   const { data: opBalance } = useQuery(
     ['balanceOf', account],
     async () => {
       if (account) {
-        return await library?.getBalance(account);
+        return await getBalance(wagmiConfig, {
+          address: account as Address,
+          chainId: wagmiChain.id,
```

Comment:
were using wagmi here?

Reply:
to get the balance for OP eth regarding for the used network. I  was able to do it with react web3